src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+public abstract class Contact {
```

Comment:
Why did you chose this to be an abstract class? Can an abstract class have implementation for methods?
Also, ensure that you add header comments.